plugins/chat/assets/javascripts/discourse/templates/admin-plugins-chat.hbs

```diff
@@ -1,3 +1,5 @@
+<AdminExportMessages />
```

Comment:
Probably need to be prefixed with Chat at least here `<Chat::Admin::ExportMessages />`

Reply:
Fixed.